generated/nifgen/nifgen.proto

```diff
@@ -165,27 +165,37 @@ service NiFgen {
 
 enum NiFgenAttributes {
   NIFGEN_ATTRIBUTE_UNSPECIFIED = 0;
+  NIFGEN_ATTRIBUTE_RANGE_CHECK = 1050002;
+  NIFGEN_ATTRIBUTE_QUERY_INSTRUMENT_STATUS = 1050003;
+  NIFGEN_ATTRIBUTE_CACHE = 1050004;
   NIFGEN_ATTRIBUTE_SIMULATE = 1050005;
+  NIFGEN_ATTRIBUTE_RECORD_COERCIONS = 1050006;
   NIFGEN_ATTRIBUTE_DRIVER_SETUP = 1050007;
-  NIFGEN_ATTRIBUTE_NUM_CHANNELS = 1050203;
+  NIFGEN_ATTRIBUTE_INTERCHANGE_CHECK = 1050021;
+  NIFGEN_ATTRIBUTE_CHANNEL_COUNT = 1050203;
+  NIFGEN_ATTRIBUTE_SPECIFIC_DRIVER_PREFIX = 1050302;
   NIFGEN_ATTRIBUTE_IO_RESOURCE_DESCRIPTOR = 1050304;
   NIFGEN_ATTRIBUTE_LOGICAL_NAME = 1050305;
   NIFGEN_ATTRIBUTE_SUPPORTED_INSTRUMENT_MODELS = 1050327;
-  NIFGEN_ATTRIBUTE_SPECIFIC_DRIVER_MAJOR_VERSION = 1050503;
-  NIFGEN_ATTRIBUTE_SPECIFIC_DRIVER_MINOR_VERSION = 1050504;
```

Comment:
these 2 attributes are obsolute, hence not present in base metadata. so they are removed